### PR TITLE
Es5

### PR DESCRIPTION
**Pull request by asimlqt**

### CODE REVIEW

File: lib/Byng/Pimcore/Elasticsearch/Processor/Element/InputElementProcessor.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    private function getNumericType($value, $default = null)
+    {
+        $val = trim($value);
+        if (is_numeric($val)) {
+            if (preg_match("/^[0-9]+$/", abs($val))) {
+                return (int) $val;
+            }
+            else {
```
Comment (ioannisGiak89) [inline lib/Byng/Pimcore/Elasticsearch/Processor/Element/InputElementProcessor.php]: else is wrapped weirdly here

File: lib/Byng/Pimcore/Elasticsearch/Processor/Page/PageProcessor.php [hunk 2]
```diff
@@ -133,6 +141,12 @@ private function processPageElement(
                 $body[$elementKey] = $this->dateElementProcessor->processElement($element);
                 return;
 
+            case "Document_Tag_Input":
+            case "Pimcore\Model\Document\Tag\Input":
+                /** @var Tag\Input $element */
+                $body[$elementKey] = $this->inputElementProcessor->processElement($element);
+                return;
+
```
Comment (ioannisGiak89) [inline lib/Byng/Pimcore/Elasticsearch/Processor/Page/PageProcessor.php:149]: Extra line here?

Comment (asimlqt) [reply lib/Byng/Pimcore/Elasticsearch/Processor/Page/PageProcessor.php:149]: and? that's how they all are

Comment (ioannisGiak89) [reply lib/Byng/Pimcore/Elasticsearch/Processor/Page/PageProcessor.php:149]: cool

**Review by ioannisGiak89 — APPROVED** [review]

**Pull request merged**